lib/todolist/tags/tag.ex

```diff
@@ -0,0 +1,18 @@
+defmodule Todolist.Tags.Tag do
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  schema "tags" do
+    field :name, :string
+
```

Comment:
aren't we missing a "has many todos" here?

Reply:
Didn't need it because on the app itself there is no functionality that required that. If we had a view tasks by tag function, then we would have needed that.
Added it in with test nevertheless. Good practice for me :)